index.html

```diff
@@ -20,16 +20,16 @@
             <div class="nav__logo-links">
                 <h3 class="nav__logo">Medical Core</h3>
                 <ul class="items">
-                    <li class="nav--item"><a class="nav__icon" href="#"></a>Home</li>
-                    <li class="nav--item"><a class="nav__icon" href="#"></a>Product</li>
-                    <li class="nav--item"><a class="nav__icon" href="#"></a>Pricing</li>
-                    <li class="nav--item"><a class="nav__icon" href="#"></a>Contact</li>
+                    <li class="items-link"><a class="items__icon" href="#"></a>Home</li>
+                    <li class="items-link"><a class="items__icon" href="#"></a>Product</li>
+                    <li class="items-link"><a class="items__icon" href="#"></a>Pricing</li>
+                    <li class="items-link"><a class="items__icon" href="#"></a>Contact</li>
```

Comment:
`items-link` improve this name block even to `item-link` and `items__icon` improve this name because this is not an icon